blackduck-security-task/task.json

```diff
@@ -157,7 +162,25 @@
       "defaultValue": false,
       "required": false,
       "helpMarkDown": "If provided, Black Duck Task will be using local network to download and execute bridge-CLI.",
-      "groupName": "bridgecliOptions"
+      "groupName": "networkOptions"
+    },
+    {
+      "name": "networkSslCertFile",
+      "type": "string",
+      "label": "SSL Certificate File",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Path to a file containing trusted CA certificates for SSL connections. This can be a PEM formatted file containing one or more certificates.",
+      "groupName": "networkOptions"
+    },
+    {
+      "name": "networkSslTrustAll",
+      "type": "boolean",
+      "label": "Trust All SSL Certificates",
+      "defaultValue": false,
+      "required": false,
+      "helpMarkDown": "Disables SSL certificate verification. Use with caution as this reduces security. When enabled, all SSL certificates will be trusted.",
```

Comment:
Message should be - TrustALL -> Disables SSL certificate verification. Use with caution.